CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+## Changed
+
+- `img` and `iframe` now default to `display: block`. Use the utility class `u-inline` if you need to replace the old behavior
```

Comment:
Wouldn't that be potentially breaking (kinda)?

Reply:
:D Yes kinda… it’s not breaking in the same way that changing a classname or variable name is though, which is why I chose the `changed` section. I’ll move it to `breaking` instead tho, to be safe